mitiq/cdr/clifford_training_data.py

```diff
@@ -43,14 +42,11 @@ def generate_training_circuits(
 ) -> List[Circuit]:
     r"""Returns a list of (near) Clifford circuits obtained by replacing (some)
     non-Clifford gates in the input circuit by Clifford gates.
-
     The way in which non-Clifford gates are selected to be replaced is
     determined by ``method_select`` and ``method_replace``.
-
     In the Clifford Data Regression (CDR) method
     :cite:`Czarnik_2021_Quantum`, data generated from these circuits is used
     as a training set to learn the effect of noise.
-
     Args:
```

Comment:
Docs tests are failing in the GitHub CI tests. Not sure why. Maybe an empty line is necessary before `Args:`.

Reply:
btw, thanks for this PR! I Iike the new file structure

Reply:
Done, no problem. First of several. Before I merge I'll let you see the new complete structure